src/frontend/components/DashboardVideoLiveControlPane/inputs/DashboardVideoLiveWidgetTextInput/index.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import { DashboardVideoLiveWidgetTextInput } from '.';
+
+let inputTextValue: string;
+let nbrOfCall = 0;
+const setValueMock = (value: string) => {
+  inputTextValue += value;
+  nbrOfCall++;
+};
+
+describe('<DashboardVideoLiveWidgetTextInput />', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+    nbrOfCall = 0;
+    inputTextValue = '';
+  });
+
+  it('renders the input with empty string and placeholder and then types some text', () => {
+    render(
+      <DashboardVideoLiveWidgetTextInput
+        placeholder="An example placeholder"
+        setValue={setValueMock}
+        title="An example title"
+        value={''}
+      />,
+    );
+
+    screen.getByPlaceholderText('An example placeholder');
+
+    const textInput = screen.getByRole('textbox', { name: 'An example title' });
+    userEvent.type(textInput, 'An example typed text');
+
+    expect(nbrOfCall).toEqual('An example typed text'.length);
+    expect(inputTextValue).toEqual('An example typed text');
+  });
+
+  it('renders the input with text', () => {
+    render(
+      <DashboardVideoLiveWidgetTextInput
+        placeholder="An example placeholder"
+        setValue={setValueMock}
+        title="An example title"
+        value={'An example typed text'}
+      />,
+    );
+
+    expect(screen.queryByText('An example placeholder')).toEqual(null);
```

Comment:
```suggestion
    screen.getByPlaceholderText('An example placeholder');
```
lol, this follows us, :-) as already explained in another test this will always be true as you don't have a text with this.
The title was indeed correct and didn't need to be changed, but you need to replace this by screen.getByPlaceholderText('An example placeholder'); to prove you have a placeholder.

Reply:
yes haha, I changed it accordingly, we should be good now :)